yt/docs/en/_includes/admin-guide/identity-sync.md

```diff
@@ -0,0 +1,17 @@
+# Import users and groups from the external systems.
+
+Manual management {{product-name}}' users and groups is discussed in the section [Managing users, groups](../../admin-guide/cluster-operations.md#managing-users,-groups-and-access-controls).  
```

Comment:
`Manual management {{product-name}}' users and groups is discussed ...` -> `{{product-name}}' users and groups manual management is discussed ...` ?